packages/websocat/build.sh

```diff
@@ -0,0 +1,20 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/vi/websocat
+TERMUX_PKG_DESCRIPTION="Command-line client for WebSockets, like netcat (or curl) for ws:// with advanced socat-like functions"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=1.11.0
+TERMUX_PKG_SRCURL=https://github.com/vi/websocat/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=943d6f66045658cca7341dd89fe1c2f5bdac62f4a3c7be40251b810bc811794f
+TERMUX_PKG_BUILD_DEPENDS="openssl"
```

Comment:
`openssl` is a runtime dependency.

Reply:
https://github.com/2096779623/termux-packages/actions/runs/4090015747/jobs/7053134230

> # openssl-sys supports OpenSSL 3 in >= 0.9.69

Reply:
Did you check the artifacts:

```bash
readelf -d $PREFIX/bin/websocat | grep NEEDED
```

Reply:
(You know what I mean?)

Reply:
> (You know what I mean?)

I may have misunderstood you before.